Add support for Python 3.9's `msg` argument to `Task.cancel()`.
